docs/pages/docs/ai-core/openai.mdx

```diff
@@ -101,6 +101,7 @@ const model = openai.chat('gpt-3.5-turbo', {
     // optional likelihood for specific tokens
     '50256': -100,
   },
+  logprobs: 0, // provides log probabilities for each token
```

Comment:
hmm I know it was my idea, just not sure how much i like this - `logprobs: 0` intuitively means "turned off". Some alternative ideas: type could be `boolean | number | undefined` where `true` means `enabled, top:0`, `false` means `disabled`, 0 means `enabled, top:0` and any positive int means `enabled, top: k`

Reply:
Yeah, I had the same concern. This is a good solution. Will update the code and comments/docs, but I think the tests are fine as they are.